hosting/src/models/event.ts

```diff
@@ -0,0 +1,32 @@
+import dayjs, { Dayjs } from 'dayjs'
+
+import { Event } from '@phits-tech/common/dist/dao-firestore/model-types'
+
+export interface EventUi {
+  slug: string
+  name: string
+  description: string
+  dateStart: Dayjs
+  dateEnd: Dayjs
+  time: string
+  location: string
+  series?: string
+  seriesType?: string
+  badges?: string[]
+  hostId?: string
+  hostName?: string
+  registrationRequired?: boolean
+  registrationUrl?: string
+}
```

Comment:
This will desync from `common`'s Event quickly. Does this work? Is it preferable?
```
type EventUi = Omit<Event, dateStart | dateEnd> & { 
  dateStart: Dayjs,
  dateEnd: Dayjs
}
```

Reply:
DONE